nextflow.config

```
@@ -196,6 +196,7 @@ profiles {
     test            { includeConfig 'conf/test.config'            }
     test_full       { includeConfig 'conf/test_full.config'       }
     test_one_sample { includeConfig 'conf/test_one_sample.config' }
+    test_sentieon   { includeConfig 'conf/test_sentieon.config'            }
```

```suggestion
    test_sentieon   { includeConfig 'conf/test_sentieon.config'   }
```

(Not that it matters to me, but I know that it does to some people ;-)  )

---

Fixed 👍🏻